docs/source/model-registry.rst

```diff
@@ -97,25 +97,37 @@ There are three programmatic ways to add a model to the registry. First, you can
 
 .. code-block:: python
 
-    from random import random, randint
+    from sklearn.datasets import make_regression
     from sklearn.ensemble import RandomForestRegressor
+    from sklearn.metrics import mean_squared_error
+    from sklearn.model_selection import train_test_split
 
     import mlflow
     import mlflow.sklearn
+    from mlflow.models.signature import infer_signature
 
-    with mlflow.start_run(run_name="YOUR_RUN_NAME") as run:
-        params = {"n_estimators": 5, "random_state": 42}
-        sk_learn_rfr = RandomForestRegressor(**params)
+    with mlflow.start_run() as run:
+        X, y = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+        X_train, X_test, y_train, y_test = train_test_split(
+            X, y, test_size=0.2, random_state=42
+        )
+        params = {"max_depth": 2, "random_state": 42}
+        model = RandomForestRegressor(**params)
+        model.fit(X_train, y_train)
+
+        # Infer the model signature
+        y_pred = model.predict(X_test)
+        signature = infer_signature(X_test, y_pred)
```

Comment:
I decided to make this a fuller example because this is the code snippet that I copy the most for trying to create a registered model and model version.

Reply:
love it!